Formula/l/lbdb.rb

```diff
@@ -21,6 +21,7 @@ class Lbdb < Formula
   end
 
   depends_on "abook"
+  depends_on "khard"
```

Comment:
Can you add a test to check that this feature is working?

Reply:
OK, will try this.

Reply:
Done, please have a look!
Quite impressed by all the GitHub Actions automation in place, nice work :)
(While I like to squash PR when merging them, I personally don't like much having to squash in a PR and force-push changes)